app/src/components/market/sections/market_buy/scalar_market_buy.tsx

```diff
@@ -173,12 +174,12 @@ export const ScalarMarketBuy = (props: Props) => {
   const currentBalance = `${formatBigNumber(collateralBalance, collateral.decimals, 5)}`
   const feeFormatted = `${formatNumber(formatBigNumber(feePaid.mul(-1), collateral.decimals))} ${collateral.symbol}`
   const baseCostFormatted = `${formatNumber(formatBigNumber(baseCost, collateral.decimals))} ${collateral.symbol}`
-  const potentialProfitFormatted = `${formatNumber(formatBigNumber(potentialProfit, collateral.decimals))} ${
-    collateral.symbol
-  }`
-  const potentialLossFormatted = `${formatNumber(formatBigNumber(potentialLoss, collateral.decimals))} ${
-    collateral.symbol
-  }`
+  const potentialProfitFormatted = `${formatNumber(
+    (Number(formatBigNumber(potentialProfit, collateral.decimals)) - totalFee).toString(),
+  )} ${collateral.symbol}`
+  const potentialLossFormatted = `${formatNumber(
+    (Number(formatBigNumber(potentialLoss, collateral.decimals)) + totalFee).toString(),
+  )} ${collateral.symbol}`
```

Comment:
looked to see if these two could be merged into one function like this 
const profitLossFormatted = (profitloss: string) => {
    return `${formatNumber(
      (
        Number(formatBigNumber(profitloss === 'profti' ? potentialProfit : potentialLoss, collateral.decimals))${profitloss === 'profti' ? -:+}
        totalFee
      ).toString(),
    )} ${collateral.symbol}`
  }
but doesn't seem like elegant solution